src/add-link.js

```diff
@@ -0,0 +1,8 @@
+document.getElementById('add-link').addEventListener('click', () => {
```

Comment:
I was trying that and it was becoming a lot of trouble for a demo. I think I will inline it in the html page so it will work without putting it into the preload-hover file.